thirdparty/kyaml/runfn/runfn.go

```diff
@@ -455,15 +404,48 @@ func getUIDGID(asCurrentUser bool, currentUser currentUserFunc) (string, error)
 	return fmt.Sprintf("%s:%s", u.Uid, u.Gid), nil
 }
 
-// ffp provides function filters
-func (r *RunFns) ffp(spec runtimeutil.FunctionSpec, api *yaml.RNode, currentUser currentUserFunc) (kio.Filter, error) {
+// getFunctionConfig will return the config that used in 'functionConfig' field in
+// function input. If r.FnConfigPath is provided, it will read and return it. Otherwise
+// returns an error.
+func (r *RunFns) getFunctionConfig() (*yaml.RNode, error) {
+	if r.FnConfigPath == "" {
+		return nil, fmt.Errorf("read function config from file but no path provided")
```

Comment:
Obligatory comment: Please audit all the error messages and make sure you're following go/kpt-errors-style-guide. I would personally not understand what this message means.

Reply:
Updated the error messages in this PR.